walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,55 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Logger;
 
+//changed from public final class to private static
 public final class FindMeetingQuery {
+  private static Logger log = Logger.getLogger("FindMeetingQuery");
```

Comment:
I think you can do something like `Logger.getLogger(FindMeetingQuery.class.name());` rather than just inputting the class name yourself. look at examples online for how to do it.